policyengine_us/parameters/gov/states/household/state_eitcs.yaml

```diff
@@ -0,0 +1,39 @@
+description: All state Earned Income Tax Credits.
+values:
+  0000-01-01:
+    - ca_eitc  # California
+    - co_eitc  # Colorado
+    - ct_eitc  # Connecticut
+    - dc_eitc  # District of Columbia
+    - de_eitc  # Delaware
+    - hi_eitc  # Hawaii
+    - il_eitc  # Illinois
+    - in_eitc  # Indiana
+    - ia_eitc  # Iowa
+    - ks_total_eitc  # Kansas - combined nonrefundable and refundable components
+    - la_eitc  # Louisiana
+    - ma_eitc  # Massachusetts
+    - md_eitc  # Maryland
+    - me_eitc  # Maine
+    - mi_eitc  # Michigan
+    - mn_wfc  # Minnesota (called “Working Family Credit”)
```

Comment:
this credit was discontinued in 2023 - we need to make a second list post 2023 for the value to not be returned

Reply:
Let's address in https://github.com/PolicyEngine/policyengine-us/issues/5441